aiaccel/optimizer/tpe_optimizer.py

```diff
@@ -182,12 +182,42 @@ def create_study(self) -> None:
         Returns:
             None
         """
-        if self.study is None:
-            self.study = optuna.create_study(
-                sampler=TPESamplerWrapper(seed=self.randseed),
-                study_name=self.study_name,
-                direction=self.config.goal.get().lower()
-            )
+
+        sampler = TPESamplerWrapper()
+        sampler._rng = self._rng
+        sampler._random_sampler._rng = self._rng
+        storage_path = str(f"sqlite:///{self.ws}/optuna-{self.study_name}.db")
+        storage = optuna.storages.RDBStorage(url=storage_path)
+        load_if_exists = self.options['resume'] is not None
+        self.study = optuna.create_study(
+            sampler=sampler,
+            storage=storage,
+            study_name=self.study_name,
+            load_if_exists=load_if_exists,
+            direction=self.config.goal.get().lower()
+        )
+
+    def resume_trial(self):
+        optuna_trials = self.study.get_trials()
+        storage_path = str(f"sqlite:///{self.ws}/optuna-{self.study_name}.db")
```

Comment:
細かいですが、これはstrになってるのでキャスト不要かと思います。
```suggestion
        storage_path = f"sqlite:///{self.ws}/optuna-{self.study_name}.db"
```

Reply:
本当ですね．直しておきます！